app/views/users/show.html.erb

```diff
@@ -1,11 +1,44 @@
 <% provide(:title, @user.name) %>
-<div class="row">
-  <aside class="col-md-4">
-    <section class="user_info">
-      <h1>
-        <%= gravatar_for @user %>
-        <%= @user.name %>
-      </h1>
-    </section>
-  </aside>
+<h1><%= t("user.profile") %></h1>
+<div class="profile">
+  <div class="row">
+    <aside class="col-md-4">
+      <section class="user_info">
+        <h1>
+          <%= gravatar_for @user %>
+          <%= @user.name %>
+        </h1>
+        <div class="date">
+          <tr>
+            <td><%= t("user.date_birth") %></td>
+            <td><%= @user.date_of_birth.strftime t("tour.fomat_date") %></td>
```

Comment:
dùng format datetime ở I18n á em, chứ ko strftime như thế này nhé 
https://guides.rubyonrails.org/i18n.html#adding-date-time-formats
https://guides.rubyonrails.org/i18n.html#how-to-store-your-custom-translations